chapter01/1.2 - Check Perm/checkPermute.js

```diff
@@ -16,4 +16,39 @@ console.log(checkPermute('aba', 'aab'), true);
 
 console.log(checkPermute('aba', 'aaba'), false);
 
-console.log(checkPermute('aba', 'aa'), false);
\ No newline at end of file
+console.log(checkPermute('aba', 'aa'), false);
+
+// Same function as a one line solution
+
+checkPermuteOneLine = (a, b) => {
+    return (a.length === b.length && (a.split("").sort().join("") === b.split("").sort().join("")))
```

Comment:
Unfortunately using inbuilt function. In most interviews, you'd be required to write that logic, unless it's just a utility, not part of the solution.